Libraries/Image/Image.windows.js

```diff
@@ -1,98 +1,65 @@
 /**
  * Copyright (c) 2015-present, Facebook, Inc.
- * All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
  *
```

Comment:
Detailed review needed.

Reply:
RNTester still working.

---
In reply to: [169649082](https://github.com/Microsoft/react-native-windows/pull/1659#discussion_r169649082) [](ancestors = 169649082)